csharp-advanced-topics/DelegatesDemo/DelegatesDemo.FuncPracticalUsage/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
As I've explained in the review (this might be too much for the article itself), but it can remain in the source code and you can point that out in the article. "For more practical examples, you can check the source code..." or something like that. But it's fine as it is since it's an evaluation article.